src/env/dev/terraform.tfvars

```diff
@@ -39,3 +40,7 @@ app_gateway_management_certificate_name = "management-dev-platform-pagopa-it"
 mock_ec_enabled = true
 
 mock_psp_enabled = true
+
+db_sku_name = "B_Gen4_1"
```

Comment:
@pasqualedevita according to [this](https://azure.microsoft.com/en-us/pricing/details/postgresql/server/), B_Gen4_1 is the cheapest postgres sku? suitable for mocks?

Reply:
should be `B_Gen5_1`

ref https://docs.microsoft.com/it-it/cli/azure/postgres/server?view=azure-cli-latest#az_postgres_server_list_skus 

`az postgres server list-skus -l westeurope`

```
[
  {
    "features": [],
    "id": "Basic",
    "maxBackupRetentionDays": 35,
    "maxLargeStorageMb": 0,
    "maxStorageMb": 1048576,
    "minBackupRetentionDays": 7,
    "minLargeStorageMb": 0,
    "minStorageMb": 5120,
    "serviceLevelObjectives": [
      {
        "edition": "Basic",
        "hardwareGeneration": "Gen5",
        "id": "B_Gen5_1",
        "maxBackupRetentionDays": 35,
        "maxStorageMb": 1048576,
        "minBackupRetentionDays": 7,
        "minStorageMb": 5120,
        "vCore": 1
      },
      {
        "edition": "Basic",
        "hardwareGeneration": "Gen5",
        "id": "B_Gen5_2",
        "maxBackupRetentionDays": 35,
        "maxStorageMb": 1048576,
        "minBackupRetentionDays": 7,
        "minStorageMb": 5120,
        "vCore": 2
      }
    ],
    "subscriptionFeatures": []
  },
  {
    "features": [
      {
        "name": "MicrosoftDBPostgreSQLBYOK",
        "value": "Enabled"
      },
      {
        "name": "MicrosoftDBPostgreSQLInfrastructureEncryption",
        "value": "Enabled"
      }
    ],
    "id": "GeneralPurpose",
    "maxBackupRetentionDays": 35,
    "maxLargeStorageMb": 0,
    "maxStorageMb": 16777216,
    "minBackupRetentionDays": 7,
    "minLargeStorageMb": 0,
    "minStorageMb": 5120,
    "serviceLevelObjectives": [
      {
        "edition": "GeneralPurpose",
        "hardwareGeneration": "Gen5",
        "id": "GP_Gen5_2",
        "maxBackupRetentionDays": 35,
        "maxStorageMb": 16777216,
        "minBackupRetentionDays": 7,
        "minStorageMb": 5120,
        "vCore": 2
      },
      {
        "edition": "GeneralPurpose",
        "hardwareGeneration": "Gen5",
        "id": "GP_Gen5_4",
        "maxBackupRetentionDays": 35,
        "maxStorageMb": 16777216,
        "minBackupRetentionDays": 7,
        "minStorageMb": 5120,
        "vCore": 4
      },
      {
        "edition": "GeneralPurpose",
        "hardwareGeneration": "Gen5",
        "id": "GP_Gen5_8",
        "maxBackupRetentionDays": 35,
        "maxStorageMb": 16777216,
        "minBackupRetentionDays": 7,
        "minStorageMb": 5120,
        "vCore": 8
      },
      {
        "edition": "GeneralPurpose",
        "hardwareGeneration": "Gen5",
        "id": "GP_Gen5_16",
        "maxBackupRetentionDays": 35,
        "maxStorageMb": 16777216,
        "minBackupRetentionDays": 7,
        "minStorageMb": 5120,
        "vCore": 16
      },
      {
        "edition": "GeneralPurpose",
        "hardwareGeneration": "Gen5",
        "id": "GP_Gen5_32",
        "maxBackupRetentionDays": 35,
        "maxStorageMb": 16777216,
        "minBackupRetentionDays": 7,
        "minStorageMb": 5120,
        "vCore": 32
      },
      {
        "edition": "GeneralPurpose",
        "hardwareGeneration": "Gen5",
        "id": "GP_Gen5_64",
        "maxBackupRetentionDays": 35,
        "maxStorageMb": 16777216,
        "minBackupRetentionDays": 7,
        "minStorageMb": 5120,
        "vCore": 64
      }
    ],
    "subscriptionFeatures": []
  },
  {
    "features": [
      {
        "name": "MicrosoftDBPostgreSQLBYOK",
        "value": "Enabled"
      },
      {
        "name": "MicrosoftDBPostgreSQLInfrastructureEncryption",
        "value": "Enabled"
      }
    ],
    "id": "MemoryOptimized",
    "maxBackupRetentionDays": 35,
    "maxLargeStorageMb": 0,
    "maxStorageMb": 16777216,
    "minBackupRetentionDays": 7,
    "minLargeStorageMb": 0,
    "minStorageMb": 5120,
    "serviceLevelObjectives": [
      {
        "edition": "MemoryOptimized",
        "hardwareGeneration": "Gen5",
        "id": "MO_Gen5_2",
        "maxBackupRetentionDays": 35,
        "maxStorageMb": 16777216,
        "minBackupRetentionDays": 7,
        "minStorageMb": 5120,
        "vCore": 2
      },
      {
        "edition": "MemoryOptimized",
        "hardwareGeneration": "Gen5",
        "id": "MO_Gen5_4",
        "maxBackupRetentionDays": 35,
        "maxStorageMb": 16777216,
        "minBackupRetentionDays": 7,
        "minStorageMb": 5120,
        "vCore": 4
      },
      {
        "edition": "MemoryOptimized",
        "hardwareGeneration": "Gen5",
        "id": "MO_Gen5_8",
        "maxBackupRetentionDays": 35,
        "maxStorageMb": 16777216,
        "minBackupRetentionDays": 7,
        "minStorageMb": 5120,
        "vCore": 8
      },
      {
        "edition": "MemoryOptimized",
        "hardwareGeneration": "Gen5",
        "id": "MO_Gen5_16",
        "maxBackupRetentionDays": 35,
        "maxStorageMb": 16777216,
        "minBackupRetentionDays": 7,
        "minStorageMb": 5120,
        "vCore": 16
      },
      {
        "edition": "MemoryOptimized",
        "hardwareGeneration": "Gen5",
        "id": "MO_Gen5_32",
        "maxBackupRetentionDays": 35,
        "maxStorageMb": 16777216,
        "minBackupRetentionDays": 7,
        "minStorageMb": 5120,
        "vCore": 32
      }
    ],
    "subscriptionFeatures": []
  }
]
```